features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml

```
@@ -97,6 +104,19 @@ repos:
       sub_dir: python/custreamz
       args: {install: *rapids_build_backend_args}
 
+- name: rapidsmpf
+  path: rapidsmpf
+  git: {<<: *git_defaults, repo: rapidsmpf, upstream: trxcllnt, tag: fix/pip-requirements}
```

Reverting now that https://github.com/rapidsai/rapidsmpf/pull/245 is merged.
```suggestion
  git: {<<: *git_defaults, repo: rapidsmpf}
```